src/Compilers/Core/CodeAnalysisTest/Emit/CustomDebugInfoReaderTests.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 extern alias PDB;
+
+
```

Comment:
That looks like a bug in the new line code of the code formatter.  Filed the following to track it:

https://github.com/dotnet/codeformatter/issues/83